pkg/scheduler/scheduler_test.go

```diff
@@ -38,7 +38,7 @@ func TestBuildDependencyGraph(t *testing.T) {
 
 	sched := New(c, nil, 0)
 
-	dg, err := sched.BuildDependencyGraph(interfaces.DependencyGraphOptions{})
+	dg, err := sched.BuildDependencyGraph(interfaces.DependencyGraphOptions{ReplicaCount: 1})
```

Comment:
Would be nice to have tests with higher ReplicaCount and incorrect ReplicaCount (is 0 incorrect?).